website/integrations/services/paperless-ng/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Paperless-ng
+---
+
+## What is Paperless-ng
+
+Modified from https://github.com/jonaswinkler/paperless-ng
+
+:::note
+Paperless-ng is an application that indexes your scanned documents and allows you to easily search for documents and store metadata alongside your documents. It was a fork from the original Paperless that is no longer maintained.
+:::
+
+:::warning
+This setup uses HTTP headers to log you in simply by providing your username as a header. Your authentik username and Paperless username MUST match. If you intend for this to be accessed externally, this requires careful setup of your reverse proxy server to not forward these headers from other sources.
+
+The author of Paperless-ng recommends you do not expose Paperless outside your network, as it was not designed for that. Instead, they "recommend that if you do want to use it, run it locally on a server in your own home."
+:::
+
+## Preparation
+
+The following placeholders will be used:
+
+- `paperless.company` is the FQDN of the Paperless-ng install.
+  
```

Comment:
Add missing authentik.company placeholder

Reply:
As it uses forward-auth, which is already covered in the documentation, there's no place where you need to put authentik.company. Should it still be listed as a placeholder if it is never referenced?

Reply:
I just checked https://goauthentik.io/integrations/services/tautulli/ for references, and that doesn't use the placeholder either but it's still there for consistency sake. Should be fine without